requirements.txt

```diff
@@ -1,4 +1,5 @@
 discord.py==2.4.0
 python-dotenv==1.0.1
 beautifulsoup4==4.12.3
-requests==2.32.3
\ No newline at end of file
+requests==2.32.3
+audioop-lts==0.2.1
```

Comment:
All looks good to me except for here. 
![image](https://github.com/user-attachments/assets/a7e92e02-85ed-4b4c-a409-bd913291adfe)
audioop is being emoved in py 3.13
Are you sure it's not accidentally picking up your other projects?

Reply:
It might, let me check because I think it had a different version selected in VSCode

Reply:
That was it, closing without merging